.github/pull_request_template.md

```
@@ -0,0 +1,93 @@
+
+
+### Root Cause (for bug fixes)
+<!-- What was causing the issue and where? -->
+
+
+
+## Trade-offs
+
+<!-- Pros, cons, and any technical debt introduced -->
+
+**Pros:**
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)